cg/services/order_validation_service/validators/data/rules.py

```diff
@@ -67,3 +71,13 @@ def validate_application_not_archived(
                 error = ApplicationArchivedError(case_name=case.name, sample_name=sample.name)
                 errors.append(error)
     return errors
+
+
+def validate_gene_panels_exist(
+    order: TomteOrder, store: Store, **kwargs
+) -> list[InvalidGenePanelsError]:
+    errors: list[InvalidGenePanelsError] = []
+    for case in order.cases:
+        case_errors: list[InvalidGenePanelsError] = validate_panels_for_case(case=case, store=store)
```

Comment:
I don't think the type hints add any value here but keep them if you think so

Reply:
I'll keep them for now, but it might be more reasonable to type hint depending on category, i.e. CaseError instead of InvalidGenePanelsError.